js/apps/admin-ui/src/realm-settings/user-profile/UnmanagedAttributesTab.tsx

```diff
@@ -0,0 +1,141 @@
+import { useTranslation } from "react-i18next";
+import { KeycloakSpinner } from "../../components/keycloak-spinner/KeycloakSpinner";
+import { useUserProfile } from "./UserProfileContext";
+import {
+  FormGroup,
+  Grid,
+  GridItem,
+  PageSection,
+  Switch,
+} from "@patternfly/react-core";
+import { HelpItem } from "ui-shared";
+import { UnmanagedAttributePolicy } from "@keycloak/keycloak-admin-client/lib/defs/userProfileMetadata";
+import { FormAccess } from "../../components/form/FormAccess";
+import { useEffect, useState } from "react";
+
+export const UnmanagedAttributesTab = () => {
+  const { config, save } = useUserProfile();
+  const { t } = useTranslation();
+  const [enableUnmanaged, setEnableUnmanaged] = useState(false);
+
+  useEffect(() => {
+    setEnableUnmanaged(config?.unmanagedAttributePolicy != undefined);
+  }, [config]);
```

Comment:
Storing this as a state, and using an effect to update it, which might cause multiple re-renders. Since the state is always derived, this can be a simple expression: 
```suggestion
  const enableUnmanaged = config?.unmanagedAttributePolicy !== undefined;
```

Also make sure to use a [strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality).

Reply:
We no longer have this tab.